diracx-logic/src/diracx/logic/auth/well_known.py

```diff
@@ -1,10 +1,29 @@
 from __future__ import annotations
 
+from importlib.metadata import PackageNotFoundError
+
+import pkg_resources
+
 from diracx.core.config.schema import Config
+from diracx.core.extensions import extensions_by_priority
 from diracx.core.models import GroupInfo, Metadata, OpenIDConfiguration
 from diracx.core.settings import AuthSettings
 
 
+def get_package_version(extension_name: str) -> dict[str, str]:
+    """Get package versions for a given diracx extension."""
+    installed_packages = pkg_resources.working_set
+    matching_packages = {}
+    for package in installed_packages:
+        if package.project_name.startswith(f"{extension_name}-"):
```

Comment:
It would be better not to rely on this convention, maybe we walk the entrypoints?